### DISCUSSION TITLE
Email & SMS Alerts

### DISCUSSION THREAD

**Discussion by anshsrtv** [AbsurdNerd/SimplifyReport_Backend#10] [Q&A]
I wanted to know which SMS service should be used for SMS alerts, as it can incur costs. Also, the email function for fire alerts has been added in #9. For other APIs, it can be added similarly. Waiting for the Maintainers for their review.

**Comment by todi-2000** [comment]
You can use django-sms-gateway package for sending sms. 
**Documentation**: https://pypi.org/project/django-sms-gateway/ for reference

**Reply by anshsrtv** [reply]
I think either the package is too old to be used or I am doing a mistake here 😝 

```
File "/home/ansh/SimplifyReport_Backend/env/lib/python3.6/site-packages/sms/models/__init__.py", line 12, in <module>
    from gateway import Gateway
ModuleNotFoundError: No module named 'gateway'
```
